Sources/Request/Request/RequestBuilder.swift

```diff
@@ -10,35 +10,26 @@ import Foundation
 @_functionBuilder
 public struct RequestBuilder {
     public static func buildBlock(_ params: RequestParam...) -> RequestParam {
-
-        let resultParams = params.filter { $0.children == nil } + params.compactMap { $0.children }.joined()
-        // Multiple Urls
-        if resultParams.filter({ $0.type == .url }).count > 1 {
-            fatalError("You cannot specify more than 1 `Url`")
-        }
-        // Missing Url
-        if resultParams.filter({ $0.type == .url }).count < 1 {
-            fatalError("You must have a `Url`")
-        }
-        return CombinedParams(children: resultParams)
+        CombinedParams(children: params)
     }
 
     public static func buildBlock(_ param: RequestParam) -> RequestParam {
-        return param
+        param
     }
 
     public static func buildIf(_ param: RequestParam?) -> RequestParam {
         if let param = param {
             return param
         }
+
         return CombinedParams(children: [])
```

Comment:
After removing `CombinedParams`, we can probably add an `EmptyParam` to replace this:
```swift
public struct EmptyParam: RequestParam {
    public func buildParam(_ request: inout URLRequest) {}
}
```